cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -34,12 +38,14 @@ public Schedule(String name, String description, String cronEntry) {
     this.name = name;
     this.description = description;
     this.cronEntry = cronEntry;
+    this.runRequirements = null;
```

Comment:
also this can cause NPE in ``RunRequirementsChecker`` when we execute ``schedule.getRunRequirements().getConcurrentProgramRunsThreshold()``, or is this constructor  not executed anywhere? seems unclear.